src/App.js

```diff
@@ -11,7 +11,7 @@ function App() {
         <Route index element={<Rockets />} />
         <Route path="rockets" element={<Rockets />} />
         <Route path="my-profile" element={<MyProfile />} />
-        <Route path="*" element={<NotFound />} />
```

Comment:
@Talha-Hanif5666 Restore the path * because that will capture all routes that are wrong and throw a 404 message